src/openassetio-core/include/openassetio/hostApi/Manager.hpp

```diff
@@ -888,20 +889,20 @@ class OPENASSETIO_CORE_EXPORT Manager final {
    *
    * @param successCallback Callback that will be called for each
    * successful default retrieved for each of the given sets in @p
-   * traitSets. It will be given the corresponding index of the traitSet
-   * in @p traitSets along with the default entity reference. The
-   * callback will be called on the same thread that initiated the call
-   * to `defaultEntityReference`.
+   * traitSets. It will be given the corresponding index of the trait
+   * set in @p traitSets along with the default entity reference. If the
+   * query is well-formed, but there simply is no default entity
```

Comment:
[consider][minor]
```suggestion
   * query is well-formed, but there is no available default entity
```
avoid editorialising.

Reply:
Updated as given in 7bc92d80